docs/design/3_1_3_programmable_logic.tex

```diff
@@ -65,7 +65,7 @@ \subsubsection{ML Hardware Accelerator Interface}
 As the accelerator's libraries are intended to process \textit{single images}, several modifications to the library were made to increase performance in video applications. In particular, high level functions were decomposed into 'initialization' and 'runtime' stages, allowing for the de-duplication of computations which are persistent across frame detections (such as the unnecessary re-uploading of model weights for each frame). These improvements resulted in a speed-up of over 2$\times$ compared to the original libraries.
 
 \subsubsection{On-Chip Interconnects}
-As seen in Figure \ref{interconnect}, the YOLOv2 accelerator is interfaced through 5 parallel AXI buses -- four buses to transfer weight data and features (including video frames) \textit{to the accelerator} and one bus to transfer system signals/results \textit{to the HPS}. A C++ program running on the HPS coordinates these interactions. In addition, each AXI bus passes through an AXI SmartConnect block (Xilinx IP) to facilitate cross-domain clocking. This is required as the YOLOv2 accelerator is driven at a higher clock frequency (150MHz compared to the provided 50MHz global clock) to further accelerate ML computations. The higher clock frequency used by the accelerator is generated through the use of a phase-locked loop (PLL) -- an efficient and commonly used construct in circuit design. The PLL is shown in Figure \ref{interconnect} as a \textit{Clocking Wizard} block.
+As seen in Figure \ref{interconnect}, the YOLOv2 accelerator is interfaced through 5 parallel AXI buses -- four buses to transfer weight data and features (including video frames) \textit{to the accelerator} and one bus to transfer system signals/results \textit{to the HPS}. A C++ program running on the HPS (\texttt{mlprog}) coordinates these interactions. In addition, each AXI bus passes through an AXI SmartConnect block (Xilinx IP) to facilitate cross-domain clocking. This is required as the YOLOv2 accelerator is driven at a higher clock frequency (150MHz compared to the provided 50MHz global clock) to further accelerate ML computations. The higher clock frequency used by the accelerator is generated through the use of a phase-locked loop (PLL) -- an efficient and commonly used construct in circuit design. The PLL is shown in Figure \ref{interconnect} as a \textit{Clocking Wizard} block.
 
 \subsubsection{HPS Operating System}
 Rather than operating on a "bare-metal" basis, a custom version of embedded Linux is used to facilitate computations and data transfer on the HPS. While an OS imposes a slight overhead, its presence greatly simplifies parallelization (required to simultaneously service requests from the PMB and acceleration core) in addition to the facilitation of debugging without the PMB attached. 
```

Comment:
```suggestion
Rather than operating on a ``bare-metal" basis, a custom version of embedded Linux is used to facilitate computations and data transfer on the HPS. While an OS imposes a slight overhead, its presence greatly simplifies parallelization (required to simultaneously service requests from the PMB and acceleration core) in addition to the facilitation of debugging without the PMB attached. 
```

It's nice that github highlights these syntax errors as bright red